### PR TITLE
JBIDE-25714 - Fuse Tooling N&N 10.3.0.AM2

### PR DESCRIPTION
**Pull request by apupier**
- Fuse version
- validation for ids
- method selection for Global bean

Signed-off-by: Aurélien Pupier <apupier@redhat.com>

### CODE REVIEW

**Comment by apupier** [comment]
@lhein @mflinn Can you review the content please?

File: documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
+= Fuse Tooling - What's New in 10.3
+:page-layout: whatsnew
+:page-component_id: fusetools
+:page-component_version: 10.3.0.AM2
+:page-product_id: jbt_core
+:page-product_version: 4.5.3.AM2
+
+== Display Fuse version corresponding to Camel version proposed
+
+When creating a new project, the Camel version can be selected among a long list. Now, the Fuse version is also displayed to help choosing version corresponding to the production version that users are using/targeting.
```
Comment (MelissaFlinn) [inline documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc]: When you create a new project, you select the Camel version from a list. Now, the list of Camel versions includes the Fuse version to help you choose the version that corresponds to your production version.

File: documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+image::./images/listOfFuseVersion.png[Fuse Version also displayed in drop-down list close to Camel version]
+
+== Update validation for similar ids between component and its definition
```
Comment (MelissaFlinn) [inline documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc]: IDs ... *a* component...

File: documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Since Camel 2.20, it is now possible to use similar ids between the component name and its definition unless the specific property "registerEndpointIdsFromRoute" is provided.
```
Comment (MelissaFlinn) [inline documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc]: Starting with Camel 2.20, you can use similar IDs for the component name and its definition ...

File: documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+The validation has been updated accordingly checking for Camel version and this specific attribute.
```
Comment (MelissaFlinn) [inline documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc]: The New Project wizard's validation process checks the Camel version and the value of the "registerEndpointIdsFromRoute" property.

File: documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+To be clear, it is in this kind o case:
```
Comment (MelissaFlinn) [inline documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc]: Replace "To be clear, it is in this kind of case"
with: "For example:"

File: documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc (hunk 4 above)
Comment (apupier) [reply documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc]: it is not the "new project wizard", it is the global validation system on camel files.
I modified to:
The validation process checks the Camel version and the value of the "registerEndpointIdsFromRoute" property.

File: documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+[source,xml]
+----
+<from id="timer" uri="timer:timerName"/>
+----
+
+== Improved guidance in method selection for factory methods on Global Bean
+
+When selecting factory method on a Global bean, a lot of possibilities were proposed in the user interface. The list provided to users contains now only the methods that are matching constraints of the Global definition kind used (Bean or Bean Factory).
```
Comment (MelissaFlinn) [inline documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc]: The list of factory methods for a global bean is now limited to only those methods that match the constraints of the bean's global definition type (bean or bean factory).

**Pull request merged**